src/api/plans.ts

```diff
@@ -36,7 +36,7 @@ router.get("/plans", async (req, res) => {
   >`SELECT user_plans.id, graduation_date, programs.name AS program_name, concentrations.name AS concentration_name
     FROM user_plans
     JOIN programs ON programs.id = program_id
-    JOIN concentrations ON concentrations.id = concentration_id
+    LEFT JOIN concentrations ON concentrations.id = concentration_id
```

Comment:
erm please fix this formatting.

Reply:
Out of scope for this PR, existing issue